daemon/src/daemon.rs

```diff
@@ -133,19 +147,43 @@ impl Document {
             // This might change in a future more peer to peer world.
             panic!("It seems {file_path} was already initialized.");
         }
+
+        // TODO: I don't love the assumption that the first document to initialize a text
```

Comment:
Maybe a bit hacky solution to this could be: What if we take the persisted initialized maps, bake them into the code and load them at each starting daemon? Then all would have the same hashes and foundation. Like a database schema that is initialized.